### PR TITLE
add freyja 1.3.7

### PR DESCRIPTION
**Pull request by kevinlibuit**
Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by kevinlibuit** [comment]
Nothing that needs to be addressed urgently, but just want to take note of this as it's relevant to this PR:
>  (optional) All test files are located in same directory as the Dockerfile (i.e. shigatyper/2.0.1/test.sh)

I wonder if there's a better practice for us here since that test dir is fully redundant in both `freyja/[1.3.4,1.3.7]` and I'd imagine it will be similarly so with other tools and test files

**Comment by kevinlibuit** [comment]
Shoot. Failed tests. Closing until I resolve this

**Comment by kevinlibuit** [comment]
@erinyoung just a heads up: the failures looked to be due to context still being set by a defined input that no longer exists (`path_to_context`). Updated this to use the parsed outputs (`./${{ steps.parse.outputs.tool }}/${{ steps.parse.outputs.version }}`)

File: freyja/1.3.7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# ARG variables only persist during build time
+ARG FREYJA_SOFTWARE_VERSION="1.3.7"
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:0.22.0"
+LABEL dockerfile.version="2"
```
Comment (kapsakcj) [inline freyja/1.3.7/Dockerfile:13]: please change dockerfile.version down to 1 since this is a new version of freyja

**Review by kapsakcj — CHANGES_REQUESTED** [review]
I'm going to request that we `wget` the files from the `/freyja/1.3.4/tests/` dir instead of duplicating.

Could you please make this change? Additionally delete the `/freyja/1.3.7/tests/` directory and files inside

**Comment by kapsakcj** [comment]
Something like this for the BAM: `wget https://github.com/StaPH-B/docker-builds/blob/master/freyja/1.3.4/tests/Freyja_WWSC2.bam?raw=true -O Freyja_WWSC2.bam`

and this for the TSVs and fasta: `wget https://raw.githubusercontent.com/StaPH-B/docker-builds/master/freyja/1.3.4/tests/Freyja_depths.tsv`

**Review by kapsakcj — APPROVED** [review]

**Comment by kapsakcj** [comment]
All looks good! I'll merge and deploy the docker image now

**Pull request merged**